controller/api/models.py

```diff
@@ -526,6 +567,21 @@ def logs(self, log_lines=str(settings.LOG_LINES)):
         data = subprocess.check_output(['tail', '-n', log_lines, path])
         return data
 
+    def logs(self, log_lines=str(settings.LOG_LINES)):
+        if self._get_logging_mode() == 'standard':
```

Comment:
not really, though that's out of scope for this PR. I think this already exists as a proposal/issue at https://github.com/deis/deis/issues/3963 to fix this hack.